src/GraphQL.MicrosoftDI.Tests/ScopedContextBase.cs

```diff
@@ -27,10 +29,11 @@ public ScopedContextBase()
             _requestServicesMock = new Mock<IServiceProvider>(MockBehavior.Strict);
             _requestServicesMock.Setup(x => x.GetService(It.Is<Type>(t => t == typeof(IServiceScopeFactory)))).Returns(serviceScopeFactory).Verifiable();
             var requestServices = _requestServicesMock.Object;
-            _scopedContext = new ResolveFieldContext
+            _scopedContext = new ResolveFieldContext<object>
```

Comment:
Why did you add generic parameter here?

Reply:
Not sure; perhaps just to match `_scopedConnectionContext`.  It works without the change so I reverted it.